core-services/prow/03_deployment/boskos_reaper.yaml

```diff
@@ -24,5 +24,5 @@ spec:
         image: gcr.io/k8s-prow/boskos/reaper:v20200427-84e5e2b2c
         args:
         - --boskos-url=http://boskos
-        - --resource-type=aws-quota-slice,azure4-quota-slice,gcp-quota-slice,openstack-quota-slice,openstack-vexxhost-quota-slice,openstack-ppc64le-quota-slice,vsphere-quota-slice,ovirt-quota-slice,ovirt-bm01,ovirt-bm02,packet-quota-slice
+        - --resource-type=aws-quota-slice,azure4-quota-slice,gcp-quota-slice,libvirt-s390x-quota-slice,openstack-quota-slice,openstack-vexxhost-quota-slice,openstack-ppc64le-quota-slice,vsphere-quota-slice,ovirt-quota-slice,ovirt-bm01,ovirt-bm02,packet-quota-slice
```

Comment:
@jaypoulz please also add the setting to `clusters/app.ci/prow/03_deployment/boskos_reaper.yaml`

Reply:
Done & rebased with updated app.ci diff